src/views/InteractiveMap/Card.vue

```diff
@@ -294,6 +326,7 @@ export default Vue.extend({
     showMethodHelpDialog: false,
     isCardDialogVisible: false,
     showDiffFVAScore: false,
+    showProteomicsData: false,
```

Comment:
No need to duplicate this field, just use a computed property with getter and setter which directly accesses the value in the store.